build/scss/scania/variables-primitive.scss

```diff
@@ -0,0 +1,111 @@
+  --scania-color-transparent-scrim-dark: rgba(0, 0, 0, 0.6);
+  --scania-color-transparent-invisible-light: rgba(255, 255, 255, 0);
+  --scania-color-transparent-invisible-dark: rgba(0, 0, 0, 0);
+  --scania-color-transparent-shadow-100: rgba(0, 0, 0, 0.1);
+  --scania-color-transparent-shadow-150: rgba(0, 0, 0, 0.15);
+  --scania-color-transparent-shadow-200: rgba(0, 0, 0, 0.2);
+  --scania-color-transparent-shadow-250: rgba(0, 0, 0, 0.25);
+  --scania-color-transparent-shadow-350: rgba(0, 0, 0, 0.35);
+  --scania-color-blue-50: #d8dde5;
+  --scania-color-blue-100: #bacde8;
+  --scania-color-blue-200: #87afe8;
+  --scania-color-blue-300: #4a89f3;
+  --scania-color-blue-400: #2a6ecf;
+  --scania-color-blue-500: #2058a8;
+  --scania-color-blue-600: #16417f;
+  --scania-color-blue-700: #0f3263;
+  --scania-color-blue-800: #041e42;
+  --scania-color-blue-900: #001533;
+  --scania-color-red-50: #ffdfe3;
+  --scania-color-red-100: #ffb9b8;
+  --scania-color-red-200: #fc8a88;
+  --scania-color-red-300: #ea4851;
+  --scania-color-red-400: #d1001b;
+  --scania-color-red-500: #b20018;
+  --scania-color-red-600: #8a0012;
+  --scania-color-red-700: #6c0011;
+  --scania-color-red-800: #480008;
+  --scania-color-red-900: #350004;
+  --scania-color-green-50: #d3eee7;
+  --scania-color-green-100: #9bd9c2;
+  --scania-color-green-200: #65bda5;
+  --scania-color-green-300: #009e7e;
+  --scania-color-green-400: #008064;
+  --scania-color-green-500: #00664f;
+  --scania-color-green-600: #004c3b;
+  --scania-color-green-700: #0a382e;
+  --scania-color-green-800: #00251b;
+  --scania-color-green-900: #001a12;
+  --scania-color-yellow-50: #f9eec3;
+  --scania-color-yellow-100: #f8e596;
+  --scania-color-yellow-200: #f4d65d;
+  --scania-color-yellow-300: #f1c21b;
+  --scania-color-yellow-400: #eaad06;
+  --scania-color-yellow-500: #b87c14;
+  --scania-color-yellow-600: #975717;
+  --scania-color-yellow-700: #6b3b10;
+  --scania-color-yellow-800: #4d260a;
+  --scania-color-yellow-900: #361b07;
+  --scania-color-grey-50: #f6f7f9;
+  --scania-color-grey-100: #edeff3;
+  --scania-color-grey-150: #e0e5eb;
+  --scania-color-grey-200: #d1d7e0;
+  --scania-color-grey-250: #b4becd;
+  --scania-color-grey-300: #93a2b7;
+  --scania-color-grey-350: #7e90a9;
+  --scania-color-grey-400: #6e829f;
+  --scania-color-grey-500: #5f728c;
+  --scania-color-grey-600: #4e5e75;
+  --scania-color-grey-650: #475467;
+  --scania-color-grey-700: #3a4554;
+  --scania-color-grey-750: #2d3643;
+  --scania-color-grey-800: #242c37;
+  --scania-color-grey-850: #1c222b;
+  --scania-color-grey-900: #15181d;
+  --scania-color-grey-950: #0e1013;
+  --scania-color-grey-1000: #000000;
+  --scania-color-grey-00: #ffffff;
+  --scania-font-family-display: scania sans headline;
+  --scania-font-family-default: scania sans;
+  --scania-font-family-semi-condensed: scania sans semi condensed;
+  --scania-font-weight-regular: 400;
+  --scania-font-weight-bold: 700;
```

Comment:
1. We need to see if these are in use in semantic level.
2. If yes, then double check if font family will work for us being all small caps - needs to be tested